core/vm/evm.go

```diff
@@ -42,11 +42,25 @@ type (
 	GetHashFunc func(uint64) common.Hash
 )
 
+// ActivePrecompiles returns the addresses of the precompiles enabled with the current
+// configuration
+func (evm *EVM) ActivePrecompiles() []common.Address {
+	switch {
+	case evm.chainRules.IsYoloV2:
+		return PrecompiledAddressesYoloV2
+	case evm.chainRules.IsIstanbul:
+		return PrecompiledAddressesIstanbul
+	case evm.chainRules.IsByzantium:
+		return PrecompiledAddressesByzantium
```

Comment:
Nit, place the last return into `default:`. Seems weird to leak out a return from the switch. You don't need to add anything at the end, Go's compiler is smart enough to realize it's an unreachable path.